src/createServer.js

```diff
@@ -1,3 +1,66 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
```

Comment:
```suggestion
const createServer = () => http.createServer((req, res) => {
```

Reply:
another variant to write it down. left this comment because saw you usually use a function declaration where better or just shorter will be better to use the arrow function)